libcodechecker/analyze/analyzers/result_handler_base.py

```diff
@@ -176,10 +176,11 @@ def analyzer_result_file(self):
             analyzed_file = self.analyzed_source_file
             _, analyzed_file_name = os.path.split(analyzed_file)
 
-            uid = str(uuid.uuid1()).split('-')[0]
+            file_name = str(self.buildaction.analyzer_type) + '_' + \
+                self.buildaction.original_command
 
-            out_file_name = str(self.buildaction.analyzer_type) + \
-                '_' + analyzed_file_name + '_' + uid + '.plist'
+            out_file_name = analyzed_file_name + '_' + \
+                hashlib.md5(file_name).hexdigest() + '.plist'
```

Comment:
Is it sufficient to only include the filename? What about files that compiled multiple times with different compilation commands? Maybe it is okay to commit it like this for now, but I would include a FIXME note here.